frontend/src/components/TodoTaskList/hook.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+import { TodoTaskProps } from '../TodoTask/TodoTask';
+
+export const useTodoTasks = (todoTasks: TodoTaskProps[]) => {
+  const [tasks, setTasks] = useState<TodoTaskProps[]>([]);
+  const [completedTaskVisible, setCompletedTaskVisible] = useState(false);
```

Comment:
Jakiego typu jest ten State? Boolean? Jeśli tak to raczej chcemy
```suggestion
  const [isCompletedTaskVisible, setIsCompletedTaskVisible] = useState<boolean>(false);
```

Reply:
setTasks to array
setCompletedTaskVisible to boolean